src/main/java/me/ryanhamshire/GriefPrevention/DeliverClaimBlocksTask.java

```diff
@@ -67,45 +65,27 @@ public void run()
         DataStore dataStore = instance.dataStore;
         PlayerData playerData = dataStore.getPlayerData(player.getUniqueId());
 
-        // check if player is idle. considered idle if
-        //    in vehicle or is in water (pushed by water)
-        //    or has not moved at least defined blocks since last check
+        // check if player is idle. Considered idle if player's yaw has not changed
         boolean isIdle = false;
-        try
-        {
-            isIdle = player.isInsideVehicle() || player.getLocation().getBlock().isLiquid() ||
-                    !(playerData.lastAfkCheckLocation == null || playerData.lastAfkCheckLocation.distanceSquared(player.getLocation()) > idleThresholdSquared);
-        }
-        catch (IllegalArgumentException ignore) //can't measure distance when to/from are different worlds
-        {
-        }
+        isIdle = !(playerData.lastAfkCheckLocation == null || playerData.lastAfkCheckLocation.getYaw() != player.getLocation().getYaw());
```

Comment:
We may want to just compare `Location#getDirection` here as that will include pitch too. Highly unlikely that a player would change one but not the other, but it's theoretically possible.

Reply:
ah, I forgot to look through all the location methods when doing this to see if there was such a direction method. Not sure what I did here, probably just started typing "get" and scrolled a little too fast perhaps.